docs/tutorials/03_ground_state_solvers.ipynb

```diff
@@ -22,21 +22,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Couldn't find cython becke routine\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/manoel/opt/anaconda3/envs/Qiskitenv/lib/python3.8/site-packages/pyscf/lib/misc.py:46: H5pyDeprecationWarning: Using default_file_mode other than 'r' is deprecated. Pass the mode to h5py.File() instead.\n",
+      "  h5py.get_config().default_file_mode = 'a'\n"
+     ]
+    }
```

Comment:
Since this h5py deprecation from within PySCF is a known issue (#248, https://github.com/pyscf/pyscf/issues/1003) I am wondering whether we should catch it (or at least locally in this tutorial)? :thinking:

Reply:
In the issue above: "This has been fixed in Pyscf v2.0.0.a, although it's not on Pypi yet."
So they already addressed it and it should be in their next release.

Reply:
I was just thinking that the tutorials could look cleaner in the meantime. v2.0.0a has been around for a while and I don't see a timeline for when 2.0.0 will be published on PyPI..

But if you think it's fine as is, then it's okay :+1: